index.html

```diff
@@ -4,7 +4,7 @@
   <title>
     Decentralized Identifiers (DIDs) v1.0
   </title>
-  <meta content='text/html; charset=utf-8' http-equiv='Content-Type'><!--
+  <meta content='text/html; charset=utf-8' http-equiv='contentType'><!--
```

Comment:
```suggestion
  <meta content='text/html; charset=utf-8' http-equiv='Content-Type'><!--
```
@rhiaro probably you didn't intend to change this line :)

Reply:
Thanks!!!